tests/telemetry/test_telemetry.py

```diff
@@ -458,6 +464,72 @@ def test_validate_entries(monkeypatch):
     assert res == (event_id, uid, action, client_time, elapsed_time)
 
 
+def test_get_latest_version(monkeypatch):
+    is_latest = telemetry.get_latest_version()
+    assert isinstance(is_latest, str)
+    version_index = [i for i, ltr in enumerate(is_latest) if ltr == '.']
+    assert len(version_index) == 2
+
+    mock_httplib = Mock()
+    mock_httplib.HTTPSConnection().request.side_effect = Exception
+    monkeypatch.setattr(telemetry, 'httplib', mock_httplib)
+    is_latest = telemetry.get_latest_version()
+    assert is_latest == ploomber.__version__
+
+    # Mock version and the conf, check it produces the same version
+
+
+def write_to_conf_file(path, version, last_check):
+    (path).write_text("stats_enabled: True\n"
+                      f"last_version_check: {last_check}\n"
+                      f"latest_ploomber_version: {version}\n")
+
+
+def test_check_version(tmp_directory, monkeypatch):
```

Comment:
can you break this test into many small tests? It's best for readability to have each test case do one thing, you can use fixtures to avoid having redundant code

Reply:
Sure.